### PR TITLE
Add SSH test case

### PR DESCRIPTION
**Pull request by hauke**
Add test case which tries to login over ssh,

### CODE REVIEW

File: tests/test_wan.py [hunk 1]
```diff
@@ -46,6 +46,8 @@ def test_download(dut):
         "Download completed ",
     )
 
+    dut.send_cmd("rm en-US", "")
```
Comment (aparcar) [inline tests/test_wan.py:49]: No need for the empty expect argument

File: tests/test_ssh.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    dut.send_cmd("uci changes", "network.lan.proto='dhcp'")
+    dut.send_cmd("uci commit")
+    dut.send_cmd("service network reload", "entered forwarding state")
+    # Wait for DHCP configuration
+    time.sleep(8)
+
+    dut.send_cmd("ip addr", "10.0.2.15/24")
+
+def test_opkg(dut):
+    configure_network(dut)
```
Comment (aparcar) [inline tests/test_ssh.py:19]: Wouldn't it be easier to log in via link local and give the LAN interface a specified MAC address? The network setup stuff should be it's own test

**Review by aparcar — CHANGES_REQUESTED** [review]

File: tests/test_ssh.py (hunk 2 above)
Comment (hauke) [reply tests/test_ssh.py:19]: This interface is not bridged to the host. The qemu command arguments we use will forward packets we send to port 8022 to port 22 of this qemu instance. This is only  virtual network we can not really access from the host.

File: tests/test_wan.py (hunk 1 above)
Comment (hauke) [reply tests/test_wan.py:49]: ok, I will remove this

**Pull request merged**